src/markdown/guide.md

```diff
@@ -25,37 +25,10 @@ curl https://api.sencrop.com/v1/ping
 ```
 ## Get your token
 
-Most of our API requires you to pass a token via the Bearer HTTP mecanism. To interact with our servers, you will need to choose a way to get one.
+If you want to use our API, please contact our API manager: hugo@sencrop.com
```

Comment:
What if Hugo is unavailable or in holidays ? Partners will have to wait to begin their integration ?

Reply:
He tried to create an alias but not working, we plan to rewrite the API documentation in the coming weeks so that's a satisfying option at the moment. 
If you have motivation, we can create an alias (api@sencrop.com if not already used) :-)

Reply:
@k-guilbert api@sencrop.com is the mailbox for api helpdesk.